### ISSUE TITLE
Undefined offset: 1

### ISSUE THREAD

**Issue by knynkwl** [mmikkel/Reasons-Craft#1]
```
2015/05/28 01:02:50 [error] [php] Undefined offset: 1 (/var/www/craft/plugins/reasons/ReasonsPlugin.php:63)
Stack trace:
#0 /var/www/craft/app/framework/base/CApplication.php(180): Craft\WebApp->processRequest()
#1 /var/www/craft/app/index.php(62): Craft\WebApp->run()
#2 /var/www/public/index.php(19): require_once()
REQUEST_URI=/admin/settings
in /var/www/craft/plugins/reasons/ReasonsPlugin.php (63)
in /var/www/craft/app/services/PluginsService.php (138)
in /var/www/craft/app/etc/web/WebApp.php (233)
2015/05/28 01:02:50 [trace] [system.CModule] Loading "errorHandler" application component
in /var/www/craft/app/etc/web/WebApp.php (571)
in /var/www/craft/plugins/reasons/ReasonsPlugin.php (63)
in /var/www/craft/app/services/PluginsService.php (138)
```

**Comment by mmikkel** [comment]
Ouch. Thx for reporting, I'll step right on it :)

**Comment by mmikkel** [comment]
I believe this is now fixed in version 0.1.1. Thanks again for the report.
